library/libGUI/lib_window_akros_main_menu.ks

```diff
@@ -0,0 +1,167 @@
+@lazyglobal off.
+
+run lib_window.
+run lib_window_menu.
+
+function open_window_akros_main_menu{
+	parameter
+		list_of_windows,
+		all_proc.
+
+	local process_state to list(
+		list(false,list_of_windows[0],"update_window_akros_main_menu",false),
+		"title_screen",abort,"child_proc_place",all_proc,list_of_windows,
+		"selected_program"
+	).
+	draw_window_akros_main_menu(process_state).
+	return process_state.
+}
+
+function draw_window_akros_main_menu{
+	parameter process_state.
+	
+	if not is_process_gui(process_state){
+		return.
+	}
+
+	local window is get_process_window(process_state).
+
+	print "aaa  k       OOOO SSSS" at(window[0]+2,window[1]+2).
+	print "  a  k       O  O S   " at(window[0]+2,window[1]+3).
+	print "aaa  k k rrr O  O SSSS" at(window[0]+2,window[1]+4).
+	print "a a  kk  r   O  O    S" at(window[0]+2,window[1]+5).
+	print "aaaa k k r   OOOO SSSS" at(window[0]+2,window[1]+6).
+
+	print "Press Abort to start. " at(window[0]+2,window[1]+8).
+
+	print "akrOS, by akrasuski1" at(window[0]+window[2]-21,
+									window[1]+window[3]-1).
+	
+	set process_state[0][3] to false. // redraw no longer needed
+}
+
+function get_process_from_name{
+	parameter
+		program_name,
+		list_of_windows,
+		list_of_processes,
```

Comment:
Why do you need parameter list_of_processes? I'd say that programs and widgets should be like plugins that are automatically installed if found in the correct place. Though I see that the task isn't trivial. Especially without string operations.